src/main/java/jenkins/plugins/publish_over_ssh/BapSshTransfer.java

```diff
@@ -42,20 +42,22 @@
     private String execCommand;
     private int execTimeout;
     private boolean usePty;
+    private boolean useAgentForwarding;
 
     BapSshTransfer(final String sourceFiles, final String remoteDirectory, final String removePrefix,
                    final boolean remoteDirectorySDF, final boolean flatten, final String execCommand, final int execTimeout) {
-        this(sourceFiles, null, remoteDirectory, removePrefix, remoteDirectorySDF, flatten, execCommand, execTimeout, false, false, false, null);
+        this(sourceFiles, null, remoteDirectory, removePrefix, remoteDirectorySDF, flatten, execCommand, execTimeout, false, false, false, false, null);
```

Comment:
Can you use a @DataBoundSetter for this instead? That is the general advisement instead of changing constructors, which may be used outside the plugin.